docs/en/user_guides/1_config.md

```diff
@@ -0,0 +1,166 @@
+        interval=1,
+        momentum=0.5**(32. / (ema_half_life * 1000.))),
+    loss_config=dict(
+        r1_loss_weight=10. / 2. * d_reg_interval,
+        r1_interval=d_reg_interval,
+        norm_mode='HWC',
+        g_reg_interval=g_reg_interval,
+        g_reg_weight=2. * g_reg_interval,
+        pl_batch_shrink=2))
+
+train_cfg = dict(max_iters=800002)  # total iterations to train the model
+
+# define optimizator for generator and discriminator
+optim_wrapper = dict(
+    generator=dict(
+        optimizer=dict(
+            type='Adam', lr=0.002 * g_reg_ratio, betas=(0,
+                                                        0.99**g_reg_ratio))),
+    discriminator=dict(
+        optimizer=dict(
+            type='Adam', lr=0.002 * d_reg_ratio, betas=(0,
+                                                        0.99**d_reg_ratio))))
+
+batch_size = 4
+data_root = './data/ffhq/images'
+
+train_dataloader = dict(
+    batch_size=batch_size,  # specify the number of samples on each GPU
+    dataset=dict(data_root=data_root))
+
+val_dataloader = dict(
+    batch_size=batch_size,  # specify the number of samples on each GPU
+    dataset=dict(data_root=data_root))
+
+test_dataloader = dict(
+    batch_size=batch_size,  # specify the number of samples on each GPU
+    dataset=dict(data_root=data_root))
+
+# add customized hooks for training
+custom_hooks = [
+    dict(
+        type='GenVisualizationHook',  # visualize training samples for GANs
+        interval=5000,  # the interval of calling this hook
+        fixed_input=True,  # whether fix the input when generate images
+        vis_kwargs_list=dict(type='GAN', name='fake_img')  # pre-defined visualize config for GAN models
+    )
+]
+
+# metrics we used to test this model
+metrics = [
+    dict(
+        type='FrechetInceptionDistance',
+        prefix='FID-Full-50k',
+        fake_nums=50000,
+        inception_style='StyleGAN',
+        inception_pkl='work_dirs/inception_pkl/ffhq256-full.pkl',  # provide the inception pkl for FID
+        sample_model='ema'  # use ema model to evaluate the metric
+    ),
+    dict(type='PrecisionAndRecall', fake_nums=50000, prefix='PR-50K'),
+    dict(type='PerceptualPathLength', fake_nums=50000, prefix='ppl-w')
+]
+default_hooks = dict(checkpoint=dict(save_best='FID-Full-50k/fid'))  # save checkpoint has the best FID metric
+
+val_evaluator = dict(metrics=metrics)  # define metric in evaluation process
+test_evaluator = dict(metrics=metrics)  # define metric in test process
+```
+
+## FAQ
+
+### Ignore some fields in the base configs
+
+Sometimes, you may set `_delete_=True` to ignore some of fields in base configs.
+You may refer to [mmcv](https://mmcv.readthedocs.io/en/latest/utils.html#inherit-from-base-config-with-ignored-fields) for simple illustration.
+
+You may have a careful look at [this tutorial](https://github.com/open-mmlab/mmengine/blob/main/docs/zh_cn/tutorials/config.md) for better understanding of this feature.
```

Comment:
This should be English docs. Refer https://github.com/open-mmlab/mmgeneration/pull/410